sdk/sqlmanagement/Azure.ResourceManager.Sql/src/Custom/MinimalTlsVersion.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Sql
+{
+    /// <summary>
+    /// A struct representing a MinimalTlsVersion
+    /// </summary>
+	public partial struct MinimalTlsVersion
+	{
+		private readonly string _value;
+
+        /// <summary> Initializes a new instance of <see cref="MinimalTlsVersion"/>. </summary>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+        public MinimalTlsVersion(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        private const string Tls_None = "None";
+        private const string Tls1_0Value = "1.0";
+        private const string Tls1_1Value = "1.1";
+        private const string Tls1_2Value = "1.2";
+        private const string Tls1_3Value = "1.3";
```

Comment:
and we could remove all of these. The generator will generate them.

Reply:
It is giving this error if I remove them: 
Build FAILED.

D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Custom\MinimalTlsVersion.cs(21,88): er
ror CS0103: The name 'Tls_None' does not exist in the current context [D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanageme
nt\Azure.ResourceManager.Sql\src\Azure.ResourceManager.Sql.csproj::TargetFramework=netstandard2.0]
D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Custom\MinimalTlsVersion.cs(21,67): er
ror CS1729: 'SqlMinimalTlsVersion' does not contain a constructor that takes 1 arguments [D:\src\NET_sdk\azure-sdk-for-
net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Azure.ResourceManager.Sql.csproj::TargetFramework=netstandard2.0]
D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Custom\MinimalTlsVersion.cs(24,87): er
ror CS0103: The name 'Tls1_0Value' does not exist in the current context [D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanag
ement\Azure.ResourceManager.Sql\src\Azure.ResourceManager.Sql.csproj::TargetFramework=netstandard2.0]
D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Custom\MinimalTlsVersion.cs(24,66): er
ror CS1729: 'SqlMinimalTlsVersion' does not contain a constructor that takes 1 arguments [D:\src\NET_sdk\azure-sdk-for-
net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Azure.ResourceManager.Sql.csproj::TargetFramework=netstandard2.0]
D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Custom\MinimalTlsVersion.cs(27,87): er
ror CS0103: The name 'Tls1_1Value' does not exist in the current context [D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanag
ement\Azure.ResourceManager.Sql\src\Azure.ResourceManager.Sql.csproj::TargetFramework=netstandard2.0]
D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Custom\MinimalTlsVersion.cs(27,66): er
ror CS1729: 'SqlMinimalTlsVersion' does not contain a constructor that takes 1 arguments [D:\src\NET_sdk\azure-sdk-for-
net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Azure.ResourceManager.Sql.csproj::TargetFramework=netstandard2.0]
D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Custom\MinimalTlsVersion.cs(30,87): er
ror CS0103: The name 'Tls1_2Value' does not exist in the current context [D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanag
ement\Azure.ResourceManager.Sql\src\Azure.ResourceManager.Sql.csproj::TargetFramework=netstandard2.0]
D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Custom\MinimalTlsVersion.cs(30,66): er
ror CS1729: 'SqlMinimalTlsVersion' does not contain a constructor that takes 1 arguments [D:\src\NET_sdk\azure-sdk-for-
net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Azure.ResourceManager.Sql.csproj::TargetFramework=netstandard2.0]
D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Custom\MinimalTlsVersion.cs(33,87): er
ror CS0103: The name 'Tls1_3Value' does not exist in the current context [D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanag
ement\Azure.ResourceManager.Sql\src\Azure.ResourceManager.Sql.csproj::TargetFramework=netstandard2.0]
D:\src\NET_sdk\azure-sdk-for-net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Custom\MinimalTlsVersion.cs(33,66): er
ror CS1729: 'SqlMinimalTlsVersion' does not contain a constructor that takes 1 arguments [D:\src\NET_sdk\azure-sdk-for-
net\sdk\sqlmanagement\Azure.ResourceManager.Sql\src\Azure.ResourceManager.Sql.csproj::TargetFramework=netstandard2.0]
    0 Warning(s)
    10 Error(s)